snuba_sdk/timeseries.py

```diff
@@ -75,7 +76,7 @@ def set_entity(self, entity: str) -> Metric:
 
 
 @dataclass
-class Timeseries:
+class Timeseries(Expression):
```

Comment:
I explictly didn't do this, because an `Expression` is assumed to be valid when it's instantiated (it runs `.validate()` in a `__post_init` function. Since Timeseries are complicated objects, I didn't want to enforce that they had to be valid immediately, so they could be constructed incrementally.

That's the reason this, `MetricsQuery` and `Query` are not instances of `Expression`.

Reply:
Makes sense, thanks for pointing this out.

Reply:
Resolved with Formula class

Reply:
Shouldn't this be removed now?